Day03/phase1 (#50)
42-AI/bootcamp_data-engineering: day03/day03.md

# Bootcamp Data Engineering

> # Day03 - Amazon Web Service

Reviewer: ```suggestion
# Day03 - Amazon Web Services
```

> Today, you will learn how to use Amazon Web Services. It is to this day, the main Cloud provider followed by Google cloud platform and Azure (for microsoft).

Reviewer: "It is to this day" is too French.
```suggestion
Today, you will learn how to use Amazon Web Services. It has become the most popular cloud service provider in the world followed by Google Cloud Platform and Microsoft's Azure.
```

> Amazon Web Services started in 2005 ad now has nearly 2000 services. Do to the large amount of services and the maturity of AWS, it is a better option to start learning to use the Cloud. 

Reviewer: ```suggestion
Amazon Web Services started in 2005 and it now delivers nearly 2000 services. Due to the large number of services and the maturity of AWS, it is a better option to start learning cloud computing.
```

> If you never heard about the Cloud, do not worry! We will go step by step to learn what the cloud is and how to use it.

Reviewer: It's better to stick to either a "you/your" or "we/our" narrative.
```suggestion
If you never heard about the Cloud before, do not worry! You will learn step by step what the Cloud is and how to use it.
```

## Notions of the day

> Create an AWS account, setup a billing alarm, 

Reviewer: ```suggestion
Create an AWS account, set up a billing alarm.
```

> The day will be divided into two parts. In the first part will learn to use a tool called terraform which will allows us to deploy/destruct an AWS infrastructure. In the second part of the day, you will learn to use a software development kit (SDK) which will allows to use python to interact with AWS.

Reviewer: It's better to stick to either a "you/your" or "we/our" narrative.
```suggestion
The day will be divided into two parts. In the first one, you will learn to use a tool called Terraform which will allow you to deploy/destruct an AWS infrastructure. In the second part of the day, you will learn to use a software development kit (SDK) which will allow you to use Python in order to interact with AWS.
```

## General rules

* The exercises are ordered from the easiest to the hardest.
* Your exercises are going to be evaluated by someone else, so make sure that your variable names and function names are appropriate and civil. 
* Your manual is the internet.
* You can also ask any question in the dedicated channel in Slack: **[42ai slack](https://42-ai.slack.com)**.
* If you find any issue or mistakes in the subject please create an issue on our dedicated repository on Github:  **[Github issues](https://github.com/42-AI/bootcamp_data-engineering/issues)**.

## Foreword

> Amazon Web Services started in 2005 ad now has nearly 2000 services. Do to the large amount of services and the maturity of AWS, it is a better option to start learning to use the Cloud. 

Reviewer: ```suggestion
Amazon Web Services started in 2005 and it now delivers nearly 2000 services. Due to the large number of services and the maturity of AWS, it is a better option to start learning cloud computing.
```

> If you never heard about the Cloud, do not worry! We will go step by step to learn what the cloud is and how to use it.

Reviewer: ```suggestion
If you never heard about the Cloud before, do not worry! You will learn step by step what the Cloud is and how to use it.
```

## Helper 

> * Your best friend for the day: **[AWS documentation](https://docs.aws.amazon.com/index.html)** and **[Terraform documentation](https://www.terraform.io/docs/index.html)**.

Reviewer: ```suggestion
* Your best friends for the day: **[AWS documentation](https://docs.aws.amazon.com/index.html)** and **[Terraform documentation](https://www.terraform.io/docs/index.html)**.
```
